### PR TITLE
refactor: auth initialization indicator and experiment cleanup

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Cleanup of auth version experiment. You can find the conclusion in the [slack message here](https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1668784914746649).
- Apply loading indicator for when the registration/login is still being initialized.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Comment by sshanzel** [comment]
Tested on `preview2.app.daily.dev` - login/registration/resend email works as intended.

File: packages/webapp/pages/signup.tsx [hunk 1]
```diff
@@ -34,7 +32,6 @@ function Signup(): ReactElement {
             trigger="login page"
             onClose={onClose}
             onSuccessfulLogin={onClose}
-            version={authVersion}
```
Comment (rebelchris) [inline packages/webapp/pages/signup.tsx:37]: Just make sure this isn't passed for tracking somewhere?

File: packages/shared/src/lib/featureValues.ts [hunk 2]
```diff
@@ -23,4 +16,6 @@ export enum ScrollOnboardingVersion {
 export enum ExperimentWinner {
   ArticleOnboarding = 'v3',
   PostCardShareVersion = 'v2',
+  AuthVersion = 'v4',
+  OnboardingVersion = 'v2',
```
Comment (rebelchris) [inline packages/shared/src/lib/featureValues.ts:20]: What is this one?

**Review by rebelchris — CHANGES_REQUESTED** [review]
Nice looks great,
Got two generic questions and one request.

Can you please tripple check the login + annonymous feed filter login flow when this hits production?

File: packages/shared/src/lib/featureValues.ts (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/lib/featureValues.ts:20]: I just saw this along the way (a magic string) and so kept it in the right place. This is the adoption of the new onboarding (v2) and removal of legacy onboarding (v1).

File: packages/webapp/pages/signup.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/pages/signup.tsx:37]: Yep. This is for layout purposes which we cleaned up. Also, I have removed the prop in the interface which typescript would show if there is any error.

**Comment by sshanzel** [comment]
> Can you please tripple check the login + annonymous feed filter login flow when this hits production?

Sure thing. I will do multiple re-runs once this hits production.

File: packages/shared/src/lib/featureValues.ts (hunk 2 above)
Comment (rebelchris) [reply packages/shared/src/lib/featureValues.ts:20]: I see ok :)
Must have been an old one indeed

**Review by rebelchris — APPROVED** [review]

**Comment by sshanzel** [comment]
@rebelchris I will merge this tomorrow morning when most of our users are still offline, and doing a lot of re-runs. Hope that is okay.

**Comment by rebelchris** [comment]
@sshanzel That's all good, thanks 🙏

**Pull request merged**